frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -1137,7 +1137,9 @@ class TableInteractive extends Component {
     } = this.props;
 
     if (!width || !height) {
-      return <div className={className} />;
+      return (
+        <div ref={el => this.props.forwardedRef(el)} className={className} />
```

Comment:
why can't we just simply pass ref?
```suggestion
        <div ref={this.props.forwardedRef} className={className} />
```